phf/src/lib.rs

```diff
@@ -5,10 +5,20 @@
 //! the documentation of those crates for more details.
 #![doc(html_root_url="https://sfackler.github.io/rust-phf/doc/v0.7.5")]
 #![warn(missing_docs)]
+#![cfg_attr(feature = "core", feature(no_std))]
+#![cfg_attr(feature = "core", no_std)]
 
 extern crate debug_builders;
 extern crate phf_shared;
 
+#[cfg(feature = "core")]
+mod std {
```

Comment:
Are these still necessary? I though the behavior of no_std had improved somewhat but could be wrong.